bcda/auth/models.go

```diff
@@ -159,8 +159,8 @@ func GenerateSystemKeyPair(systemID uint) (string, error) {
 
 	return string(privateKeyBytes), nil
 }
-func CreateAlphaToken(ttl int, acoSize string) (string, error) {
-	aco, err := createAlphaEntities(acoSize)
+func CreateAlphaToken(ttl int, acoCMSID string) (string, error) {
```

Comment:
I think we've replaced enough of `CreateAlphaToken`'s functionality to remove it now.  

Either way, because it's called by the [`tokengen` script](https://github.com/CMSgov/bcda-ops/blob/6052e9b26d09924993e02df0ca633cf4df464060/scripts/tokengen#L65), which in turn is used by a couple of playbooks, we'll want an ops ticket to either change the call or remove the `tokengen` script.